src/BookNavigator/bookmarks/ia-bookmarks.js

```diff
@@ -113,9 +120,18 @@ class IABookmarks extends LitElement {
     // eslint-disable-next-line
     this.defaultColor = this.bookmarkColors[0];
     this.api = api;
+    this.deleteModalConfig = new ModalConfig({
+      title: 'Delete Bookmark',
+      headline: 'This bookmark contains a note. Deleting it will permanently delete the note. Are you sure?',
+      headerColor: '#194880',
+    });
   }
 
-  updated() {
+  updated(changed) {
+    if (changed.has('options')) {
```

Comment:
It looks like you got rid of the `options` property. Will this ever fire?

Reply:
thanks for catching, updated to check for `displayMode`